docs/SPEC.md

```diff
@@ -2610,7 +2610,7 @@ from(bucket: "telegraf/autogen")
 ```
 #### Distinct
 
-Distinct produces the unique values for a given column.
+Distinct produces the unique values for a given column. Null values in the given column are dropped.
```

Comment:
This differs from SQL behavior which will put a null in the output if there was one or more nulls in the column.  I thought that @nathanielc said in Slack thread that we should keep a single null...

Reply:
I didn't see his comment. I'll fix that.

Reply:
Fixed.